.github/workflows/ESlint.yaml

```
@@ -0,0 +1,47 @@
+name: ESlint
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+
+jobs:
+  code:
+    name: Versões NodeJS
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version:
+          - 14.x
+          - 15.x
+
+    steps:
+      - name: Obtendo versão mais nova do Codigo
+        uses: actions/checkout@v2
+
+      - name: Configurando NodeJS ${{ matrix.node-version }}
+        uses: actions/setup-node@main
```

```suggestion
        uses: actions/setup-node@v1
```

É má prática não fixar versão.